src/api/api.js

```diff
@@ -0,0 +1,31 @@
+
+const apiURL = "http://api.giphy.com";
+const apiTrends = "/v1/gifs/trending";
+const apiSearch = "/v1/gifs/search";
+const apiKey = "MKGAXNQl5cXUBSBMrXSsufVZ9bqvhX6p"
```

Comment:
Use `.env` variables to store your api keys, also it is a good practice to use CAPS for static vars. Dont version that file and provide an example for its usage.

https://www.npmjs.com/package/dotenv